localization/english/wc_holy_orders_l_english.yml

```diff
@@ -406,7 +406,7 @@
  holy_order_firstborn_elementalists_article:1 "the "
  holy_order_firstborn_elementalists:1 "Firstborn Elementalists"
  holy_order_earth_wind_and_fire_article:0 "the "
- holy_order_earth_wind_and_fire:0 "Earth, Wind and Fire"
+ holy_order_earth_wind_and_fire:0 "Storm, Earth and Fire"
```

Comment:
```suggestion
 holy_order_earth_wind_and_fire:0 "Storm, Earth, and Fire"
```

Reply:
Shit, yeah, I somehow have to get that into my brain, to add the , before and ^^'